freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/flow/FreeIpaPostInstallService.java

```diff
@@ -47,7 +48,9 @@ public class FreeIpaPostInstallService {
 
     private static final String SET_PASSWORD_EXPIRATION_PERMISSION = "Set Password Expiration";
 
-    private static final int MAX_USERNAME_LENGTH = 255;
+    private static final int MAX_NAME_LENGTH = 255;
+
+    private static final int MAX_DOMAIN_PART_LENGTH = 64;
```

Comment:
this is the hostname part actually

Reply:
But that's true for every part of a domain name

Reply:
hmm, ok, in that case fine, but the domain in FMS world usually means the part after the hostname. so maybe MAX_FQDN_PART_LENGTH in this case would be a bit better and more obvious

Reply:
OK, maybe I need to rename it because it just 63 chars + the delimiter :)

> Domain name syntax, internationalization
The definitive descriptions of the rules for forming domain names appear in RFC 1035, RFC 1123, RFC 2181, and RFC 5892. A [domain name](https://en.wikipedia.org/wiki/Domain_name) consists of one or more parts, technically called labels, that are conventionally [concatenated](https://en.wikipedia.org/wiki/Concatenated), and delimited by dots, such as example.com.
>The right-most label conveys the [top-level domain](https://en.wikipedia.org/wiki/Top-level_domain); for example, the domain name www.example.com belongs to the top-level domain com.
>The hierarchy of domains descends from right to left; each label to the left specifies a subdivision, or [subdomain](https://en.wikipedia.org/wiki/Subdomain) of the domain to the right. For example, the label example specifies a subdomain of the com domain, and www is a subdomain of example.com. This tree of subdivisions may have up to 127 levels.[[26]](https://en.wikipedia.org/wiki/Domain_Name_System#cite_note-26)
> A label may contain zero to 63 characters.